pkg/cmd/extension/command_test.go

```diff
@@ -286,6 +286,49 @@ func TestNewCmdExtension(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "installing local extension without executable with TTY shows warning",
+			args: []string{"install", "."},
+			managerStubs: func(em *extensions.ExtensionManagerMock) func(*testing.T) {
+				em.InstallLocalFunc = func(dir string) error {
+					return &ErrExtensionExecutableNotFound{
+						Dir:  tempDir,
+						Name: "gh-test",
+					}
+				}
+				em.ListFunc = func() []extensions.Extension {
+					return []extensions.Extension{}
+				}
+				return nil
+			},
+			wantStderr: fmt.Sprintf(
+				"! %s",
```

Comment:
I would like for this test to be checking the hardcoded string directly, rather than relying on the implementation detail of `Error()`. This way, we have confidence that if the returned value from `Error()` was changed, something would tell us.

Reply:
Resolved by ec4b2dfe1b6770845874bd78433c843083e0e2b3